modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala

```diff
@@ -108,7 +108,7 @@ class UpdateService[F[_]](
               ignUpdate =>
                 update.groupId == ignUpdate.groupId && update.nextVersion == ignUpdate.nextVersion &&
                   ignUpdate.artifactIds.exists(id => update.artifactId.startsWith(id))
-            )
+          )
         )
```

Comment:
If you run `sbt scalafmt` it should reformat the code such that `sbt scalafmtCheck` is happy.